spec/rubocop/cop/rails/negate_include_spec.rb

```diff
@@ -12,6 +12,38 @@
     RUBY
   end
 
+  it 'registers an offense and corrects when using `!include?` without receiver' do
+    expect_offense(<<~RUBY)
+      def foo
+        !include?(2)
+        ^^^^^^^^^^^^ Use `.exclude?` and remove the negation part.
+      end
+
+      def self.foo
+        !include?(2)
+        ^^^^^^^^^^^^ Use `.exclude?` and remove the negation part.
+      end
```

Comment:
That's what I noticed after reconsidering. It looks better to allow if there is no receiver because there may be user-defined `include?`, which may not have `exclude?`.

And can you separate the tests for `def` and `defs`?

Reply:
Updated to not raise an offense when there is no receiver. Now this PR is tiny 👍.
And that was actually an overkill from my side to detect cases like `!include?` inside `def exclude?`.